src/Store/InputStoreConnector.re

```diff
@@ -128,8 +128,9 @@ let start = (window: option(Revery.Window.t), runEffects) => {
 
     let getValue = v =>
       switch (Hashtbl.find_opt(currentConditions, v)) {
-      | Some(variableValue) => variableValue
-      | None => false
+      | Some(true) => WhenExpr.Value.True
+      | Some(false)
+      | None => WhenExpr.Value.False
       };
```

Comment:
We'll need to generalize this as well, to keep track of the contexts globally, and also be able to extend it contextually.

Reply:
That makes sense. It seems like we might want to factor out this whole function: https://github.com/onivim/oni2/blob/21f0fffb5c9343a7a6b11840c3bd43da053175fc/src/Store/InputStoreConnector.re#L19 into a common place - something that gives us the 'context' for a `State.t`, and then we can use it to evaluate expressions anywhere.

Reply:
Yeah, I'm currently working on an overhaul of the command infrastructure, which would use `WhenExpr.t` as well to determine if it's enabled, so will do this as part of that.